src/components/landing/ScrollControl.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+interface ScrollContainerProps {
+  slideNumber: number;
+  setSlideNumber: (slideNumber: number) => void;
+}
+
+function ScrollContainer(props: ScrollContainerProps) {
+  const { slideNumber, setSlideNumber } = props;
+  const slideNumberList = [1, 2, 3, 4, 5, 6];
+
+  return (
+    <StScrollContainer>
+      {slideNumberList.map((number) => {
+        return (
+          <li
+            key={number}
+            className={slideNumber === number ? 'isActive' : 'unActive'}
+            onClick={() => setSlideNumber(number)}></li>
+        );
```

Comment:
이렇게 하면 className 사용하지 않아도 됩니다 ㅎㅎ

```suggestion
        return (
          <StListItem
            key={number}
            isActive={slideNumber === number}
            onClick={() => setSlideNumber(number)}></StListItem>
        );
```